src/components/Login.tsx

```diff
@@ -1,19 +1,24 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import { useAuth } from "../context/useAuth";
+import "../styles/login.css";
 
 export type LoginData = {
   email: string;
   password: string;
 };
 
-const Login = () => {
+type LoginProps = {
+  setUserMenu: (arg: boolean) => void;
```

Comment:
I would change this to a more generic function. For example onLogin, here we can pass a callback that contains more logic that needs to be run when the user is logged in. Now we pass setUserMenu that doesn't really have to do with the login function.